pvlib/pvsystem.py

```diff
@@ -831,7 +834,7 @@ def physicaliam(aoi, n=1.526, K=4., L=0.002):
 
```

Comment:
I implemented your suggestion except for setting negative angles to np.nan, to keep consistency with the other IAM functions for now...